extensions/2.0/Vendor/EXT_lights_image_based/README.md

```diff
@@ -0,0 +1,136 @@
+                "rotation": [0, 0, 0, 1],
+                "irradianceCoefficients": [...3 x 9 array of floats...],
+                "specularImageSize": 256,
+                "specularImages": [
+                    [... 6 cube faces for mip 0 ...],
+                    [... 6 cube faces for mip 1 ...],
+                    ...
+                    [... 6 cube faces for mip n ...]
+                ],
+            }
+        ]
+    }
+}
+```
+
+## Prefiltered Specular Radiance Cubemaps
+
+The cubemap used for specular radiance is defined along with its prefiltered mipmaps with each face of the cube defined as separate images. The various mip levels evenly map to roughness values from 0 to 1 in the PBR material and should be generated with a principled multi-scatter GGX normal distribution. The data in the maps represents illuminance in lux (lm/m^2).
+
+The entire mip chain of images should not be provided. Instead, the lowest-resolution mip should have sufficient size to represent the maximally-blurred radiance map (say, 16x16) corresponding to roughness=1. The `specularImageSize` value defines the largest dimension of mip 0 and, taken together with the number of defined mips, should give the loading runtime the information it needs to generate the remainder of the mip chain and sample the appropriate mip level in the shader.
+
+Cube faces are defined in the following order and adhere to the standard orientations as shown in the image bellow.
+1. Positive X
+1. Negative X
+1. Positive Y
+1. Negative Y
+1. Positive Z
+1. Negative Z
+
+<figure>
+<img src="./figures/Cube_map.svg"/>
+<figcaption><em>Cube map orientation reference.<br>Image by <a href="//commons.wikimedia.org/w/index.php?title=User:Microwerx&amp;action=edit&amp;redlink=1" class="new" title="User:Microwerx (page does not exist)">Microwerx</a> - <span class="int-own-work" lang="en">Own work</span>, <a href="https://creativecommons.org/licenses/by-sa/4.0" title="Creative Commons Attribution-Share Alike 4.0">CC BY-SA 4.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=48935423">Link</a></em></figcaption>
+</figure>
+
+Note that for this extension, each saved image must be flipped about its vertical axis to correspond with the way <a href="https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#images">glTF references texture space</a>.
+
+https://en.wikipedia.org/wiki/Cube_mapping
+
+
+## Irradiance Coefficients
+
+This extension uses spherical harmonic coefficients to define irradiance used for diffuse lighting. Coefficients are calculated for the first 3 SH bands (l=2) and take the form of a 9x3 array.
+https://metashapes.com/blog/realtime-image-based-lighting-using-spherical-harmonics/
+https://cseweb.ucsd.edu/~ravir/papers/envmap/envmap.pdf
+http://graphics.stanford.edu/papers/envmap/
+
+Generating SH Coefficients
+http://graphics.stanford.edu/papers/envmap/prefilter.c
+
+Using SH Coefficients
+http://graphics.stanford.edu/papers/envmap/envmaprender.in
+
```

Comment:
GitHub strips one-time line breaks, so these links aren't formatted properly. Consider wrapping them with titles like:

- [Realtime Image Based Lighting using Spherical Harmonics](https://metashapes.com/blog/realtime-image-based-lighting-using-spherical-harmonics/)
- [An Efficient Representation for Irradiance Environment Maps (PDF)](https://cseweb.ucsd.edu/~ravir/papers/envmap/envmap.pdf)
...